source/docs/contributing/watching-changes.rst

```diff
@@ -0,0 +1,44 @@
+# Watching changes
+
+This document describes different ways to stay up-to-date on the changes being made to the WPILib suite.
+
+## Stages of development
+
+To provide context about the different ways to stay up to date, below are the relevant stages of development:
+
+- Discord: The [Unofficial FIRST Robotics Competition Discord Server](https://discord.com/invite/frc) contains a programming discussion channel where people bring up their initial ideas and then receive feedback on them, either fleshing them out or learning why they might not be good ideas. Most of the major changes will have some discussion in here, but not all.
+- GitHub issue: Issues on the relevant GitHub repository are the next step toward a change being implemented. If the idea has gone through Discord, then this simply presents the fleshed out idea; Otherwise, this presents the idea and sometimes get initial discussion. Sometimes changes will skip this step and move straight to a pull request.
```

Comment:
nit: assuming this is meant to be an ordered list, saying this is the next step is superfluous

Probably needs a comparison to when something is ready for a GitHub issue (if ever? Changes can go from discord to PR, just like they can first pop up in PR). Github issues provide a single place for all discussion about a topic. Useful when the discussion spans a long period of time, or history of the discussion is useful

Reply:
Fair point that "next step" is superfluous, and that reminds me that I should probably make the ordered nature more apparent. Maybe I'll change it to be a numbered list, but I'm hesitant since that could imply that everything needs to happen when that isn't the case. Maybe instead a wording change?

I agree that the wording should be clearer about the difference between GitHub issues and Discord conversations, but I do want to point out is that the stages of development are provided "To provide context about the different ways to stay up to date" (so the target audience is an observer), not as a guide for how someone should develop an idea.